exporter/datadogexporter/README.md

```diff
@@ -72,3 +72,7 @@ To get the service field correctly populated in your logs, you can specify servi
 [alpha]:https://github.com/open-telemetry/opentelemetry-collector#alpha
 [contrib]:https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
 [AWS]:https://aws-otel.github.io/docs/partners/datadog
+
+### How to add custom log source
+
+In order to add a custom source to your OTLP logs, set log attribute or resource attribute `datadog.log.source`.
```

Comment:
It's unusual to support an attribute both as a log attribute and a resource attribute (I don't know of any examples from semantic conventions at least). Why are we doing it like this?

Reply:
I agree with you, and ideally this would be read only from resource attributes. It's currently done this way due to how the logs code is structured. The origin is accessible [here](https://github.com/DataDog/datadog-agent/blob/main/comp/otelcol/otlp/components/exporter/logsagentexporter/logs_exporter.go#L80), and this is where we need to set the [source](https://github.com/DataDog/datadog-agent/blob/main/comp/otelcol/otlp/components/exporter/logsagentexporter/logs_exporter.go#L84). 

At this point, the attributes and resource attributes have been merged and are not distinguishable within AdditionalProperties. See [Attributes](https://github.com/DataDog/opentelemetry-mapping-go/blob/main/pkg/otlp/logs/transform.go#L120-L124)/ [resource attributes](https://github.com/DataDog/opentelemetry-mapping-go/blob/main/pkg/otlp/logs/transform.go#L128-L137).

We have the same issue with [status](https://github.com/DataDog/datadog-agent/blob/main/comp/otelcol/otlp/components/exporter/logsagentexporter/logs_exporter.go#L76). This is fixable, but requires a refactor of the logs code to allow this.

Reply:
As discussed offline, will create a card to refactor logs path, and in the meantime will remove support for log attributes from docs.